cmake/Modules/FindSDL2.cmake

```diff
@@ -1,17 +1,16 @@
+# Copyright (C) 2015 Ryan Pavlik <ryan.pavlik@gmail.com> <abiryan@ryand.net>
+# Copyright (C) 2015 Sensics, Inc.
+# Copyright (C) 2019 - 2021 Settlers Freaks <sf-team at siedler25.org>
+#
+# SPDX-License-Identifier: BSL-1.0
+
 # - Find SDL2
 # Find the SDL2 headers and libraries
 #
 #  SDL2::SDL2 - Imported target to use for building a library
 #  SDL2::SDL2main - Imported interface target to use if you want SDL2main.
 #  SDL2_FOUND - True if SDL2 was found.
 #  SDL2_DYNAMIC - If we found a DLL version of SDL (meaning you might want to copy a DLL from SDL2::SDL2)
-#
```

Comment:
we probably should not change these, since they're from the original file like that